src/main/java/repository/CourseRepository.java

```diff
@@ -0,0 +1,5 @@
+package repository;
+
+public class CourseRepository {
```

Comment:
This class should an interface. The real implementation will be provided by the Mongo class